codebuild_specs/e2e_workflow.yml

```diff
@@ -135,24 +135,24 @@ batch:
           CLI_REGION: us-east-1
       depend-on:
         - publish_to_local_registry
-    - identifier: api_1_resolvers_sync_query_datastore_api_6
+    - identifier: rds_pg_refersTo_api_1_resolvers_sync_query_datastore
```

Comment:
This will go away once my other commit shows up on Github. I pushed it but its not there yet for some reason.